test/Tests/Execution/SchemaExecutionTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using GraphQL;
+using GraphQL.Conventions.Tests;
+using GraphQL.Conventions.Tests.Templates.Extensions;
+
+namespace Tests.Execution
+{
+    public class SchemaExecutionTests
+    {
+        [Test]
+        public void Can_Have_Decimals_In_Schema()
+        {
+            var schema = SchemaBuilderHelpers.Schema<SchemaTypeWithDecimal>();
```

Comment:
Without the refactoring done further down, you can simply have `SchemaExecutionTests` derive from `ConstructionTestBase` and use `Schema<SchemaTypeWithDecimal>()` here.